### DISCUSSION TITLE
Is there any way to limit access to loki datasources in grafana open source

### DISCUSSION THREAD

**Discussion by ahmedTouati** [grafana/grafana#43979] [Data Sources]
To achieve a complete multitenancy solution in loki, each user must have only access to his datasource which streams the logs related to that user only. As I know restricting access to datasources is only available in grafana enterprise edition. So is there any other way so that a specific user can only select his datasource or whene selecting a different datasource no logs must be available?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
